### PR TITLE
Adding new sector page

### PR DESCRIPTION
**Pull request by Aymen-Khanfir**
Updating application architecture and adding new sector page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +28,7 @@
     "@radix-ui/react-scroll-area": "^1.2.1",
     "@radix-ui/react-slot": "^1.1.0",
     "@radix-ui/react-switch": "^1.1.1",
+    "@radix-ui/react-tabs": "^1.1.2",
```
Comment (aymenkhanfir55) [inline package.json:31]: Remove this dependencies cause we don't need it anymore

File: src/components/ui/tabs.tsx
Comment (aymenkhanfir55) [inline src/components/ui/tabs.tsx:1]: remove it as well

File: package.json (hunk 1 above)
Comment (aymenkhanfir55) [inline package.json:31]: Remove this cause we don't need it anymore

File: .github/workflows/auto-pr.yml
Comment (aymenkhanfir55) [inline .github/workflows/auto-pr.yml:1]: Redo this changes please

**Review by aymenkhanfir55 — CHANGES_REQUESTED** [review]
Please make sure to resolve this issues

**Review by Aymen-Khanfir — COMMENTED** [review]
reviewed all

**Review by aymenkhanfir55 — APPROVED** [review]

**Review by aymenkhanfir55 — APPROVED** [review]

**Pull request merged**